error: expected ';' before 'typedef'

In file included from gdt.h:4:
types.h:4:30: error: expected ';' before 'typedef'
    4 | typedef unsigned char uint8_t
      |                              ^
      |                              ;
    5 |
    6 | typedef short int16_t;
      | ~~~~~~~

--- a/src/include/types.h
+++ b/src/include/types.h
@@ -1,7 +1,7 @@
 #ifndef TYPES_h
 #define TYPES_h
 typedef char int8_t;
-typedef unsigned char uint8_t
+typedef unsigned char uint8_t;
 
 typedef short int16_t;
 typedef unsigned short uint16_t;

